### ISSUE TITLE
Update translations for 4.1.2

### ISSUE THREAD

**Issue by andergmartins** [publishpress/PublishPress-Future#1047]
Hi @wocmultimedia please, could you update the translations for the attached package?

[post-expirator-4.1.2-beta.2.zip](https://github.com/user-attachments/files/17836678/post-expirator-4.1.2-beta.2.zip)

**Comment by wocmultimedia** [comment]
Hi @andergmartins 
Here they are the translation updates
[TranslationUpdates_post-expirator-4.1.2-beta.2.zip](https://github.com/user-attachments/files/17837329/TranslationUpdates_post-expirator-4.1.2-beta.2.zip)

**Comment by andergmartins** [comment]
Hi Angelo, thanks!

**Comment by wocmultimedia** [comment]
Hi @andergmartins ,
I downloaded and took in charge the translations. I will send it to you later today.

**Comment by wocmultimedia** [comment]
Hi @andergmartins 
I checked the ES-FR-IT translations and updated. Take note that the publishpress-future-pro-future_workflow_editor_script.pot is not present in the languages folder, so I just checked and corrected the publishpress-future-pro.po files related to their .pot file.
[LanguagesFolderUpdatedTranslations_Future-Pro-4.1.2-beta.2.zip](https://github.com/user-attachments/files/17849862/LanguagesFolderUpdatedTranslations_Future-Pro-4.1.2-beta.2.zip)

Feel free to contact me for any inconvenience.
